Formula/arpack.rb

```diff
@@ -0,0 +1,56 @@
+class Arpack < Formula
+  desc "Routines to solve large scale eigenvalue problems"
+  homepage "https://github.com/opencollab/arpack-ng"
+  url "https://github.com/opencollab/arpack-ng/archive/3.5.0.tar.gz"
+  sha256 "50f7a3e3aec2e08e732a487919262238f8504c3ef927246ec3495617dde81239"
+  head "https://github.com/opencollab/arpack-ng.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  depends_on :fortran
+  depends_on "veclibfort"
+  depends_on :mpi => [:optional, :f77]
+
+  def install
+    args = [ "--disable-dependency-tracking",
```

Comment:
I'm not sure how to quote the last argument, with the space in it, if I used `%W[]` :(